src/core/amount.js

```diff
@@ -614,10 +586,12 @@ function(quality, counterCurrency, counterIssuer, opts) {
     quality as stored :  5 USD          /  3000000 drops
     inverted          :  3000000 drops  /          5 USD
   */
-  var adjusted = opts.inverse ? inverse(value) : value;
-  var nativeAdjusted = adjusted;
+  const valueStr = mantissa.toString() + 'e' + offset.toString();
+  let nativeAdjusted = this._is_native ?
+    new XRPValue(valueStr) : new IOUValue(valueStr);
```

Comment:
@clark800 Re your most recent comment: this is why I am passing an IOUValue instance to the constructor of XRPValue.